test/helpers/params/constructParams.js

```diff
@@ -111,43 +131,255 @@ const createClassParam = (
   vestingDuration
 ) => [className, classCap, individualCap, vestingCliff, vestingDuration];
 
-function getClassParams(params) {
-  if (!params) params = {};
+/**
+ *
+ * @param {GetClassParamsFromTypeParams} params
+ * @returns {[className: string, classCap: string, individualCap: string, vestingCliff: number, vestingDuration: number]}
+ */
+function getClassParamsFromType(params = {}) {
   if (!params.fundingTokenDecimal) params.fundingTokenDecimal = 12;
-  if (!params.class) params.class = classTypes.CLASS_1;
+  if (!params.class) params.class = classTypes.CLASS_DEFAULT;
   const getFundingAmounts = getTokenAmount(params.fundingTokenDecimal);
 
   switch (params.class) {
     case classTypes.CLASS_1:
```

Comment:
Descriptive names for classes (instead of just `Class_1`) will be more helpful in the long run.
Cf. different Seed "constellations" and description: https://www.notion.so/primedao/Fixtures-Creating-the-Perfect-Seed-and-LBP-tobeadded-8436d9275d3a4065803674543ca037bd#90a8446d1935433b9ddc7f7dbed4c08d